distributionviewer/core/static/css/login.styl

```diff
@@ -6,6 +8,11 @@
 
   p {
     text-align: center;
+
+  }
+  p.auth-help {
```

Comment:
Not a fan of adding the node type to the target since it adds specificity and is less OOCSSy (can't believe I just wrote that). Just `.auth-help` is fine.